### PR TITLE
Fixes dependency error (see issue #56 and #57) with the latest activesupport

### PR DESCRIPTION
**Pull request by HorlogeSkynet**
Hello,

This PR should fixes the #57. **EDIT: and the #56 !**
I've cloned, changed the ActiveSupport version according to [their latest stable version](https://rubygems.org/gems/activesupport/versions/4.2.9), built and installed this Gem ; It works well.

Cya 👋

### CODE REVIEW

File: jemoji.gemspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
   s.name        = 'jemoji'
   s.summary     = 'GitHub-flavored emoji plugin for Jekyll'
-  s.version     = '0.8.0'
+  s.version     = '0.8.1'
```
Comment (parkr) [inline jemoji.gemspec]: We do this. Please revert this change.

Comment (HorlogeSkynet) [reply jemoji.gemspec]: I don't have the branch anymore, feel free to commit the same change and delete this PR, it'll be OK
